### ISSUE TITLE
tls1.h Present But Cannot Be Compiled

### ISSUE THREAD

**Issue by vedranmiletic** [troglobit/inadyn#126]
On Fedora 23, x86_64 with GCC 5.3.1 and OpenSSL 1.0.2h-fips.

```
checking openssl/tls1.h usability... no
checking openssl/tls1.h presence... yes
configure: WARNING: openssl/tls1.h: present but cannot be compiled
configure: WARNING: openssl/tls1.h:     check for missing prerequisite headers?
configure: WARNING: openssl/tls1.h: see the Autoconf documentation
configure: WARNING: openssl/tls1.h:     section "Present But Cannot Be Compiled"
configure: WARNING: openssl/tls1.h: proceeding with the compiler's result
configure: WARNING:     ## --------------------------------------------------------- ##
configure: WARNING:     ## Report this to https://github.com/troglobit/inadyn/issues ##
configure: WARNING:     ## --------------------------------------------------------- ##
checking for openssl/tls1.h... no
```

Is this expected?

**Comment by troglobit** [comment]
How do you call the configure script?  I've never tested on Fedora, but it should work.  Have you tried the GnuTLS support instead?

**Comment by vedranmiletic** [comment]
Yes, that was my first try. It does not find libgnutls-openssl. I have installed gnutls-devel.

**Comment by troglobit** [comment]
Yeah that was a Debian-only backwards-compat thing they forward ported for a while, sorry about that.  I hope the GIT master version of inadyn will work better for you!

**Comment by troglobit** [comment]
Should be fixed as of v2.0-rc1, please reopen should the problem still be present.
